src/Frontend/Themes/Bootstrap/src/Layout/Templates/UserTemplates/quote.html

```diff
@@ -1,15 +1,9 @@
-<blockquote>
-  <div class="row">
-    <div class="col-sm-2">
-      <img src="http://placehold.it/128x128" alt="Placeholder" class="img-responsive img-circle" data-ft-label="Avatar" data-ft-type="image" />
-    </div>
-    <div class="col-sm-10">
-      <p data-ft-type="textarea" data-ft-label="Quote text">The only way to do great work is to love what you do</p>
-      <footer>
-        <span data-ft-type="text" data-ft-label="author">Steve Jobs</span>,
-        <span data-ft-type="text" data-ft-label="function">Founder</span> of
-        <a href="http://www.apple.com" data-ft-type="link" data-ft-label="Website">Apple</a>
-      </footer>
-    </div>
+<div class='row blog-quote'>
+  <div class='col-12 quote-body'>
+    <div class='quote-icon'><i class='fa fa-quote-right'></i></div>
+    <figure>
+      <div class='blockquote'><p class='quote-text' data-ft-label="Quote tekst" data-ft-type="textarea">Numquam porro quaerat quidem quos sit ullam voluptas voluptatum?</p></div>
+      <figcaption><span data-ft-label="Quote caption" data-ft-type="text">Voornaam Naam, Bedrijfsnaam </span><cite data-ft-label="Quote citation" data-ft-type="text" data-ft-block-optional="true">- Tevreden klant</cite></figcaption>
```

Comment:
Waarom de `figcaption`? Er is toch geen image?  Zou je geen image toevoegen, of is het meestal zonder image?

Reply:
Er is meestal geen image, maar de quote is de figure en figcaption is het correcte element voor de caption
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

Reply:
Ok